pkg/scheduler/core/spreadconstraint/group_clusters.go

```diff
@@ -130,7 +133,42 @@ func (info *GroupClustersInfo) generateClustersInfo(clustersScore framework.Clus
 	sortClusters(info.Clusters)
 }
 
-func (info *GroupClustersInfo) generateZoneInfo(spreadConstraints []policyv1alpha1.SpreadConstraint) {
+func (info *GroupClustersInfo) calcGroupScore(clusters []ClusterDetailInfo, rbSpec *workv1alpha2.ResourceBindingSpec) int64 {
+	// First, calculate the base score.
+	var highScoreSum int64
+	var selectedNum int
```

Comment:
Can you explain the reason why this algorithm is done in this way?

Reply:
Hi @whitewindmills, thanks for your hard work, can we divide the improvement of the calculation score into a separate pr? Let's focus on the reconstruction selection algorithm part so that the code review and integration can be accelerated.

Reply:
Cool! It's a good idea. I'll do that.